packages/@okta/vuepress-theme-prose/components/Sidebar.vue

```diff
@@ -4,77 +4,116 @@
       <SidebarItem v-for="(link, key) in navigation" :key="key" :link="link" />
     </ul>
   </aside>
-
 </template>
 
 <script>
-  import _ from 'lodash';
-  export default {
-    name: 'Sidebar',
-    components: {
-      SidebarItem: () => import('../components/SidebarItem.vue'),
-    },
-    data() {
-      return {
-        sidebarActive: false,
-        usingFile: false
+import _ from "lodash";
+import { getGuidesInfo, guideFromPath } from "../util/guides";
+
+export default {
+  name: "Sidebar",
```

Comment:
Is this a good opportunity to rename this component? It is not a sidebar, its visual position is dynamic and not semantic. Semantically, this is tree navigation.

Reply:
true I can rename it to 'TreeNav' in next commit. : )

Reply:
On a second thought we'll have to address this in new pull request. Changes would overwhelm this pull request as name 'SideBar' is being used in close to 8 different vue files and scss files.

Reply:
I believe that sidebar here is fine, as the idea of what this does is a `Sidebar`.  using that term does not define the location of it.

Reply:
Haha totally fine to leave, but `Sidebar` is 100% about a location – a bar to the side. The term defines location, not functionality.

In narrow viewports, this is not in a sidebar. In the future, tree navigation could be shown in more and various locations. So a semantic term would be better than an incorrect presentational term.